<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1091</width>
    <height>625</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(252, 175, 62);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>100</y>
      <width>539</width>
      <height>335</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(238, 238, 236);
border-radius: 25px;
</string>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>150</y>
       <width>67</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>ID</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>210</y>
       <width>67</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>PW</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="login_id">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>150</y>
       <width>113</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-style: solid;
border-width: 2px;
color: black;
</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="login_pw">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>210</y>
       <width>113</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-style: solid;
border-width: 2px;
color: black;
</string>
     </property>
    </widget>
    <widget class="QPushButton" name="login_btn">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>150</y>
       <width>89</width>
       <height>81</height>
      </rect>
     </property>
     <property name="palette">
      <palette>
       <active>
        <colorrole role="Button">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>136</red>
           <green>138</green>
           <blue>133</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>136</red>
           <green>138</green>
           <blue>133</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>136</red>
           <green>138</green>
           <blue>133</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="Button">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>136</red>
           <green>138</green>
           <blue>133</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>136</red>
           <green>138</green>
           <blue>133</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>136</red>
           <green>138</green>
           <blue>133</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="Button">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>136</red>
           <green>138</green>
           <blue>133</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>136</red>
           <green>138</green>
           <blue>133</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>136</red>
           <green>138</green>
           <blue>133</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background-color: rgb(136, 138, 133);
border-style: solid;
border-radius: 20px
}
QPushButton:hover{
background-color: rgb(180, 255, 255);
border-style: solid;
border-radius: 20px}

QPushButton:pressed{
background-color: rgb(100, 255, 255);
border-style: solid;
border-radius: 20px}
</string>
     </property>
     <property name="text">
      <string>Login</string>
     </property>
    </widget>
    <widget class="QPushButton" name="sing_up_btn">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>250</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background-color: rgb(136, 138, 133);
border-style: solid;
border-radius: 5px
}
QPushButton:hover{
background-color: rgb(180, 255, 255);
border-style: solid;
border-radius: 5px}

QPushButton:pressed{
background-color: rgb(100, 255, 255);
border-style: solid;
border-radius: 5px}
</string>
     </property>
     <property name="text">
      <string>sign_up</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>60</y>
       <width>281</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>교수용 로그인</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="sing_widget" native="true">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>90</y>
      <width>581</width>
      <height>381</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(238, 238, 236);
border-radius: 25px;
</string>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>120</y>
       <width>67</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>이름</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>160</y>
       <width>67</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>ID</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="name">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>120</y>
       <width>113</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-style: solid;
border-width: 2px;
color: black;
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="sing_di">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>160</y>
       <width>113</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-style: solid;
border-width: 2px;
color: black;
</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>30</y>
       <width>281</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>회원가입</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="sing_pw">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>200</y>
       <width>113</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-style: solid;
border-width: 2px;
color: black;
</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>200</y>
       <width>67</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>PW</string>
     </property>
    </widget>
    <widget class="QPushButton" name="confirm_btn">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>280</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background-color: rgb(136, 138, 133);
border-style: solid;
border-radius: 20px
}
QPushButton:hover{
background-color: rgb(180, 255, 255);
border-style: solid;
border-radius: 20px}

QPushButton:pressed{
background-color: rgb(100, 255, 255);
border-style: solid;
border-radius: 20px}
</string>
     </property>
     <property name="text">
      <string>확인</string>
     </property>
    </widget>
    <widget class="QPushButton" name="back_btn">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>280</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background-color: rgb(136, 138, 133);
border-style: solid;
border-radius: 20px
}
QPushButton:hover{
background-color: rgb(180, 255, 255);
border-style: solid;
border-radius: 20px}

QPushButton:pressed{
background-color: rgb(100, 255, 255);
border-style: solid;
border-radius: 20px}
</string>
     </property>
     <property name="text">
      <string>취소</string>
     </property>
    </widget>
    <widget class="QPushButton" name="idcheck_btn">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>160</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background-color: rgb(136, 138, 133);
border-style: solid;
border-radius: 20px
}
QPushButton:hover{
background-color: rgb(180, 255, 255);
border-style: solid;
border-radius: 20px}

QPushButton:pressed{
background-color: rgb(100, 255, 255);
border-style: solid;
border-radius: 20px}
</string>
     </property>
     <property name="text">
      <string>중복확인</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="sing_pw_2">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>240</y>
       <width>113</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-style: solid;
border-width: 2px;
color: black;
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>240</y>
       <width>67</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>PW 확인</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="menu_widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>1091</width>
      <height>511</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(238, 238, 236);</string>
    </property>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1091</width>
       <height>511</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="2">
       <widget class="QWidget" name="widget_4" native="true">
        <widget class="QPushButton" name="pushButton_2">
         <property name="geometry">
          <rect>
           <x>-10</x>
           <y>0</y>
           <width>551</width>
           <height>261</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>40</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Quiz Into</string>
         </property>
        </widget>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QWidget" name="widget_3" native="true">
        <widget class="QPushButton" name="pushButton">
         <property name="geometry">
          <rect>
           <x>-30</x>
           <y>0</y>
           <width>581</width>
           <height>261</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>40</pointsize>
          </font>
         </property>
         <property name="acceptDrops">
          <bool>false</bool>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="inputMethodHints">
          <set>Qt::ImhNone</set>
         </property>
         <property name="text">
          <string>분석</string>
         </property>
         <property name="autoRepeat">
          <bool>false</bool>
         </property>
         <property name="autoExclusive">
          <bool>false</bool>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QWidget" name="widget_6" native="true">
        <widget class="QPushButton" name="pushButton_4">
         <property name="geometry">
          <rect>
           <x>-10</x>
           <y>-10</y>
           <width>551</width>
           <height>261</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>40</pointsize>
          </font>
         </property>
         <property name="text">
          <string>상담</string>
         </property>
        </widget>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QWidget" name="widget_5" native="true">
        <widget class="QPushButton" name="pushButton_3">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>-10</y>
           <width>551</width>
           <height>261</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>40</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Q/A</string>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1091</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
